Application/EdFi.Ods.Features/Controllers/OpenApiMetadataController.cs

```diff
@@ -32,11 +31,11 @@ namespace EdFi.Ods.Features.Controllers
     [AllowAnonymous]
     [ApplyOdsRouteRootTemplate]
     [Route("metadata")]
+    [Route("{tenantId}/metadata")]
```

Comment:
this attribute is not needed; that route template is covered by OdsRouteRootTemplateConvention